configs/template_config.yaml

```diff
@@ -59,35 +92,20 @@ activation_dropout: 0.0
 batch_size: 8
 # Checkpoints (bool): Whether to save checkpoints during training
 checkpoints: false
-# Device Configuration (str)
-device: "cuda" # Choices: "cpu", "cuda"
 # Dropout (float): Probability of an element to be zeroed during dropout
 dropout: 0.1
 # Embedding Dimension (int): Embedding dimension size of each token
 embed_dim: 80
-# Epochs (int): Number of epochs to train for
-epochs: 1
 # FFN Dimension (int): Hidden layer size of Feed Forward Network (FFN)
 ffn_dim: 12
-# Fully Sharded Data Parallel (bool): Whether to shard Module parameters across
-# data parallel workers or not (with the FairScale library)
-fsdp: false
```

Comment:
The normal `train_model.py` script no longer runs because `fsdp` and `validation_frequency` are no longer included in the config file. How are we expecting to resolve this?

Reply:
I will add them back in. However, we should consider whether or not we want to maintain both `train_model.py` and `train_model_lightning.py`, because the latter should be able to do everything the former is doing, while enabling multi-device training.

Reply:
I think that, as you mention, we should just drop the original `train_model.py` script in favor of Lightning. Thoughts @JayOrten and @KimballNJardine?

Reply:
I'm okay with that